commit 3c0ac8bf2a0b38e6402c24418b47c685d40f998f (refs/changes/64/5364/3)
Author: Alex Wilson <alex.wilson@joyent.com>
Date:   2019-01-15T00:03:04+00:00 (9 months ago)
    
    TRITON-1087 cnsadm broken after TRITON-1044 jsprim update
    Reviewed by: Cody Peter Mello <cody.mello@joyent.com>

diff --git a/bin/cnsadm b/bin/cnsadm
index ebc7a40..0b60a26 100755
--- a/bin/cnsadm
+++ b/bin/cnsadm
@@ -143,7 +143,7 @@ sapi.get('/configs/' + zoneName, function (err, req, res, obj) {
 
 	Object.keys(config).forEach(function (k) {
 		if (obj.metadata.hasOwnProperty(k)) {
-			if (!jsprim.deepEquals(config[k], obj.metadata[k]))
+			if (!jsprim.deepEqual(config[k], obj.metadata[k]))
 				waiting[k] = true;
 			config[k] = obj.metadata[k];
 		}
diff --git a/lib/cmd-utils.js b/lib/cmd-utils.js
index 57b398c..e6e60a3 100644
--- a/lib/cmd-utils.js
+++ b/lib/cmd-utils.js
@@ -102,7 +102,7 @@ function parseModifiers(args, oldObj, schema) {
 			for (i = 0; i < orig.length; ++i) {
 				var add = true;
 				for (var j = 0; j < val.length; ++j) {
-					if (jsprim.deepEquals(
+					if (jsprim.deepEqual(
 					    orig[i], val[j])) {
 						add = false;
 						break;
